.github/workflows/pypi.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish to PyPI
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags: [ 'v*' ]
```

Comment:
This will run on pushes to `master` OR tag pushes *separately*. You probably wanted to publish releases only on tags. If you want each push to master to make a release, this will need more work, like a mechanism for generating unique intermediate dev versions.